src/django/api/migrations/0195_add_event_index.py

```
@@ -0,0 +1,22 @@
+        migrations.AlterModelOptions(
+            name="event",
+            options={
+                "indexes": [
+                    models.Index(
+                        fields=["content_type", "object_id"],
+                        name="api_event_content_object_idx",
+                    )
+                ]
+            },
+        ),
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)